packages/skeleton-react/src/components/Progress/types.ts

```diff
@@ -0,0 +1,36 @@
+export interface ProgressProps {
+	/** Set the value */
+	value?: number;
+	/** Set the minimum value */
+	min?: number;
+	/** Set the maximum value */
+	max?: number;
+	/** Set the aria-labelledby */
+	ariaLabelledby?: string;
+
+	// Root ---
+	/** Set root base classes */
+	base?: string;
+	/** Set root background classes  */
+	bg?: string;
+	/** Set root height classes */
+	height?: string;
+	/** Set root rounded classes */
+	rounded?: string;
+	/** Set root arbitrary classes */
+	classes?: string;
+
+	// Meter ---
+	/** Set meter base classes. */
+	meterBase?: string;
+	/** Set meter bg classes */
+	meterBg?: string;
+	/** Set meter rounded classes. */
+	meterRounded?: string;
+	/** Set meter transition classes. */
+	meterTransition?: string;
+	/** Set meter indeterminate animation classes */
```

Comment:
I'd suggest a slightly longer description here where we indicate the inherent animation is embedded within the component. Not sure how we word that exactly though to be succinct.

Reply:
Added a longer name, could you take a look again?